src/main/steps/c100-rebuild/permission/template.njk

```diff
@@ -0,0 +1,19 @@
+{% extends "common/template.njk" %}
+
+{% block page_content %}
+
+    <h1 class="govuk-heading-xl">{{title}}</h1>
+    <p class="govuk-body">{{courtPermissionLine1}}</p>
+    <p class="govuk-body">{{courtPermissionLine2}}</p>
+    <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-7 govuk-!-margin-bottom-6">
+        <li>{{courtPermissionBullet1}}</li>
```

Comment:
Same as below comment. Please use an array for these common bullet points & use a for loop to render them.

Reply:
completed